src/modules/acbs/dto/acbs-create-deal-guarantee.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { DateString } from '@ukef/helpers/date-string.type';
+
+export interface AcbsCreateDealGuaranteeDto {
```

Comment:
is this a DTO? or an interface?

Reply:
Do you mean that a DTO can only be a class? I consider it a DTO in the sense that these are the fields for the object we use to transfer data to ACBS